### ISSUE TITLE
can you edit it so it can mine any block?

### ISSUE THREAD

**Issue by user632** [LXYan2333/Fabric-Bedrock-Miner#6]

**Comment by LXYan2333** [comment]
Will do in the next version.

**Comment by krispyking24** [comment]
i get that u are probably really busy can u try and give an eta of when the update will come out

**Comment by JacktheDevel0per** [comment]
Yes. It's Kari block check might see if LXYan wants to add it as a list with frames and stuff like that or make a toggle with Malilib..

**Comment by LXYan2333** [comment]
won't have enough spare time until July 2022 (probably earlier if I can finish my paper faster).

**Comment by Anonymous17532** [comment]
It would be cool if you could get this to work on paper and bukkit servers but I don't think that's possible

**Comment by MrRedRhino** [comment]
It is working on spigot, so I guess it works on craftbukkit too, but paper doesn't work since it has a config option to completely disable permanent-block-breaking and also one to disable piston replacement

**Comment by fooeyround** [comment]
For headless pistons it does vigorous checks for pistons without head and stuff it probably causes some lag but them skipping a game tick might make up for all of it.

**Comment by erxson** [comment]
hi

**Comment by probablypablito** [comment]
i made a pull request a while ago that added that feature, though it got closed. its on 1.17.1 as well

**Comment by zdwnss** [comment]
> 

呐 所以现在还没有(哭出来
